draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -2244,38 +2308,41 @@ Internet-Draft                 ACTN POI                      March 2022
    no impact on the YANG models nor on the interoperability at the MPI,
    but cares have to be taken to avoid missing information.
 
-5.3. SR-TE Path Setup and Update
+5.3. TE Path Setup and Update
 
-   This version of the draft assumes that SR-TE path setup and update at
+   This version of the draft assumes that TE path setup and update at
    the MPI could be done using the generic TE tunnel YANG data model,
-   defined in [TE-TUNNEL], with SR-TE specific augmentations, as also
-   outlined in section 1 of [TE-TUNNEL].
+   defined in [TE-TUNNEL], with technology-specific agumentations,
+   described in section 3.2.3.
 
-   When a new SR-TE path needs to be setup, the MDSC can use the [TE-
-   TUNNEL] model to request the P-PNC to setup TE paths, properly
-   specifying the path constraints, such as the explicit path, to force
-   the P-PNC to setup an SR-TE path that meets the end-to-end TE and
-   biding constraints and uses the optical tunnels setup by the MDSC for
-   the purpose of supporting this new SR-TE path.
 
-   The [TE-TUNNEL] model supports requesting the setup of both end-
-   to-end as well as segment TE tunnels (within one domain).
 
 
+Peruzzini et al.      Expires December 29, 2022               [Page 43]
+
+Internet-Draft                 ACTN POI                       June 2022
 
 
-Peruzzini et al.      Expires September 7, 2022               [Page 42]
-
-Internet-Draft                 ACTN POI                      March 2022
+   When a new TE path needs to be setup, the MDSC can use the [TE-
+   TUNNEL] model to request the P-PNC to set it up, properly specifying
+   the path constraints, such as the explicit path, to force the P-PNC
+   to setup an TE path that meets the end-to-end TE and biding
```

Comment:
"biding" should be "binding"

Reply:
Fixed with https://github.com/FabioPeruzzini/actn-poi/pull/94/commits/db7de1b347f717eebe690317827570fbf6e8970b